manifest_test.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2016 The Go Authors. All rights reserved.
+// Copyright 2016-2017 The Go Authors. All rights reserved.
```

Comment:
The standard practice for this in golang/go seems to be to just put in the year from when the file was created, not update it. Let's follow that pattern.